birdhouse/config/canarie-api/docker_configuration.py.template

```diff
@@ -256,8 +256,8 @@ PLATFORMS = {
         'redirect': {
             'doc': 'https://pavics-sdi.readthedocs.io/en/latest/arch/backend.html',
             'releasenotes': 'https://github.com/bird-house/birdhouse-deploy/releases',
-            'support': 'https://github.com/Ouranosinc/pavics-sdi/issues',
-            'source': 'https://github.com/Ouranosinc/pavics-sdi',
+            'support': '${SUPPORT_URL}',
+            'source': 'https://github.com/bird-house/birdhouse-deploy',
```

Comment:
I would like the feedback also. Maybe @dbyrns also has some insights on that. 
Feels weird that the "source" of the platform points to the docs repo though.

Reply:
The section in question is "platform" which mean the frontend for Canarie. It used to be the react frontend, but now the notebooks are really the frontend, so pavics-sdi make sense I think. @fmigneault, if you look above you'll see a node section which is the backend and is correctly using birdhouse-deploy as source.

Reply:
I see. In that case, I will revert both of above `support` and `source` to the original values. 
Does it also apply to `PLATFORMS['pavics']['info']` elements as well? 

How should another organization define their configuration? 
The way I see it now, it's that every server will identify itself as the same `pavics` platform definition. 
Is this valid considering any organization can customize it with different features enabled/disabled?
In other words, is that definition still valid as the *default* template shared and not organization-specific?

Reply:
> Does it also apply to PLATFORMS['pavics']['info'] elements as well?

Yes, PLATFORMS['pavics'] apply to the frontend, while SERVICES['node'|'indices'|...] apply to backend core or services

> How should another organization define their configuration?

I think @tlvu is using the canarie api for monitoring with the an optional component (https://github.com/bird-house/birdhouse-deploy/tree/master/birdhouse/optional-components/canarie-api-full-monitoring), but for other organization, the canarie API itself is completely irrelevant. A standard nginx could be used there and move the whole canarie-api image as an optional component (https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/docker-compose.yml#L4)

Anyway, this is out of scope right now and this whole config could be empty for someone not requiring it.

Reply:
I reverted all env-var metadata utils I initially added because it seems too much trouble to change without blocking this PR... 
Opened another issue #159.

I still strongly believe that this config is plain wrong. 
It makes senses only when deployed to `pavics.ouranos.ca`, and all other instances report erroneous references. 
But since there is much opposition to proposed changes, until then, it looks like this will remain the prod config of Ouranos server.

Reply:
In fact, like you said, this whole config is so much linked to Ouranos and because other organization could have a completely different setup (using optional components), the easiest way would be to force each deployment to define its own config file and set the location in an environment variable here : https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/docker-compose.yml#L22. The current template file would then be turned into an example file.

Reply:
> I think @tlvu is using the canarie api for monitoring with the an optional component (https://github.com/bird-house/birdhouse-deploy/tree/master/birdhouse/optional-components/canarie-api-full-monitoring),

Exact.

>  but for other organization, the canarie API itself is completely irrelevant. 

Exact, but they can also leverage the monitoring side, like Ouranos does.

All other "description/information" links are indeed completely irrelevant so they can be "wrong" without impact.

> A standard nginx could be used there and move the whole canarie-api image as an optional component (https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/docker-compose.yml#L4)

Agreed.  Currently, nginx and the canarie monitoring cronjob and web interface https://github.com/Ouranosinc/CanarieAPI/blob/73835095262093b2c67e14a0659e8874b5720794/Dockerfile#L32-L35 are all running in the same `proxy` container.

With the current optional component mechanism, they can be splitted.